Formula/l/lld.rb

```diff
@@ -0,0 +1,65 @@
+class Lld < Formula
+  desc "LLVM Project Linker"
+  homepage "https://lld.llvm.org/"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-19.1.0/llvm-project-19.1.0.src.tar.xz"
+  sha256 "5042522b49945bc560ff9206f25fb87980a9b89b914193ca00d961511ff0673c"
+  # The LLVM Project is under the Apache License v2.0 with LLVM Exceptions
+  license "Apache-2.0" => { with: "LLVM-exception" }
+  head "https://github.com/llvm/llvm-project.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(/^llvmorg[._-]v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "cmake" => :build
+  depends_on "llvm"
+  depends_on "zstd"
+  uses_from_macos "zlib"
+
+  def install
+    system "cmake", "-S", "lld", "-B", "build",
+                    "-DBUILD_SHARED_LIBS=ON",
+                    "-DCMAKE_INSTALL_RPATH=#{rpath}",
+                    "-DLLD_BUILT_STANDALONE=ON",
+                    "-DLLD_VENDOR=#{tap&.user}",
+                    "-DLLVM_ENABLE_LTO=ON",
+                    "-DLLVM_INCLUDE_TESTS=OFF",
+                    *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    bin.each_child do |child|
+      next if child.basename.to_s == "lld"
+
+      child.unlink
+      bin.install_symlink "lld" => child.basename
+    end
```

Comment:
Odd that this is needed. Maybe `-DLLVM_USE_SYMLINKS=ON`? Should have been default, but might not work if building standalone with existing `llvm` installation.

Reply:
Yea, wondered that too, since `llvm` had symlinks for `lld` but this doesn't. Will experiment with `LLVM_USE_SYMLINKS`